### ISSUE TITLE
has-valid-accessibility-role lacks several valid accessibility roles

### ISSUE THREAD

**Issue by AryanJ-NYC** [FormidableLabs/eslint-plugin-react-native-a11y#53]
# Actual
`has-valid-accessibility-role` only allows `accessibilityRole` to be one of:
  * adjustable
  * button
  * header
  * image
  * imagebutton
  * keyboardkey
  * link
  * none
  * search
  * summary

when React Native supports 27 `accessibilityRole`s: https://facebook.github.io/react-native/docs/accessibility#accessibilityrole-ios-android

# Expected
`has-valid-accessibility-role` allows all 27 `accessibilityRole`s found on https://facebook.github.io/react-native/docs/accessibility#accessibilityrole-ios-android

By the way, thanks for this package!

Oh! And I'll be more than happy to help out with this pull request once this is confirmed to be a bug and not a feature.
